### ISSUE TITLE
Create definition of done

### ISSUE THREAD

**Issue by phildree** [amosproj/amos2023ss01-apache-pulsar-ui#51]
## User story
1. As a Team
2. I we need an understandable and realistic definition of done
3. So that we can decide whether a ticket is done or not

## Acceptance criteria
* [ ] Add definition of done to planning docs

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
